Doc/conf_api.py

```diff
@@ -30,14 +30,18 @@
 @author: Gael Goret and Eric C. Pellegrini
 '''
 
+import sphinx
+
+major, minor, _, _, _ = sphinx.version_info
+
 # -- General configuration -----------------------------------------------------
 
 # If your documentation needs a minimal Sphinx version, state it here.
 needs_sphinx = '1.0'
 
 # Add any Sphinx extension module names here, as strings. They can be extensions
 # coming with Sphinx (named 'sphinx.ext.*') or your custom ones.
-extensions = ['sphinx.ext.autodoc', 'sphinx.ext.viewcode', 'sphinx.ext.graphviz', 'sphinx.ext.inheritance_diagram', 'sphinx.ext.pngmath']#,'rst2pdf.pdfbuilder']
+extensions = ['sphinx.ext.autodoc', 'sphinx.ext.viewcode', 'sphinx.ext.graphviz', 'sphinx.ext.inheritance_diagram''sphinx.ext.pngmath' if minor <= 6 else 'sphinx.ext.imgmath']
```

Comment:
I think you are missing a comma here. Regardless, your changes caused a lot of warnings that it cannot create inheritance diagrams when running `build_api`.